deploy/31_tenderly.ts

```diff
@@ -12,12 +14,57 @@ function mapFullKeyToShortKey(networks: Networks, fullKey: string) {
 }
 
 const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  /*
   const currentNetworkType: NetworkType = networks[hre.network.name].type;
   if (currentNetworkType == NetworkType.local) {
     hre.deployments.log('Not verifying contracts on localhost networks.');
     return;
   }
 
+  const accounts = await hre.ethers.getSigners();
```

Comment:
was this causing you issues? should we reenable?

Reply:
Yep it was causing issues. Until we can standarize and test this tenderly implementation out, and also add a simple enable/disable toggle, I will keep it disabled.